server/routes/referrals/serviceUserDetailsPresenter.ts

```diff
@@ -0,0 +1,21 @@
+import { ServiceUser } from '../../services/interventionsService'
+import { SummaryListItem } from '../../utils/viewUtils'
+
+export default class ServiceUserDetailsPresenter {
+  constructor(private readonly serviceUser: ServiceUser) {}
+
+  readonly title = `${this.serviceUser.firstName || 'Service user'}'s information`
+
+  readonly summary: SummaryListItem[] = [
+    { key: 'CRN', lines: [this.serviceUser.crn], isList: false },
+    { key: 'Title', lines: [this.serviceUser.title || ''], isList: false },
```

Comment:
I think that, although in this case, the outcome will be the same, it's a bit clearer to use the [nullish coalescing operator `??`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) instead of `||`, since `||` tests truthiness (e.g. is the string non-empty) but I think we just care whether it's `null` or not.

Reply:
Yeah, I keep seeing you using this operator but hadn't taken enough time to look into it fully, have changed it.